src/main/scala/cromwell/engine/package.scala

```diff
@@ -14,15 +14,27 @@ import cromwell.binding.values.WdlValue
  *
  * Internally, this package is built on top of [[cromwell.binding]].
  */
-
 package object engine {
-  type WorkflowId = UUID
+
```

Comment:
Is there structure to these? i.e. could the `fooID` stuff (please be `fooId` when this is merged) live under a single parent? Could the abort function stuff live under some structure? 

I'm not suggesting that changes be made, if you don't think that's the case so be it